src/util/Conversions.h

```diff
@@ -48,6 +49,9 @@ inline string convertFloatToIndexWord(const string& value,
 //! Converts like this: "PP0*2E0*1234 to "12.34 and M-0*1E9*876 to -0.123".
 inline string convertIndexWordToFloat(const string& indexWord);
 
+//! Converts like this: "PP0*2E0*1234 to "12.34 and M-0*1E9*876 to -0.123".
+inline float convertIndexWordToFloatValue(const string& indexWord);
```

Comment:
why is there a `string convertIndexWordToFloat(const string& indexWord);` wit the same comment but returning a `std::string`? If I'm seeing this right the other one is just for displaying as a string. So for the sake of better names I propose dropping the suffix `…Value` from this one and adding `…String` to the other one or alternatively I think we could drop the other one entirely and rely on later conversion to `string`

Reply:
Originaly there was only the one converting the index word to float, which is used when generating the json output. As SUM and AVG need to interpret floats I added the convert to float value version. I renamed them to convertIndexWordToFloatString and convertIndexWordToFloat.